Altis_Life.Altis/core/pmenu/fn_giveItem.sqf

```diff
@@ -26,12 +26,12 @@ if (isNull _unit) exitWith {ctrlShow[2002,true];};
 //A series of checks *ugh*
 if (!([_val] call TON_fnc_isnumber)) exitWith {hint localize "STR_NOTF_notNumberFormat";ctrlShow[2002,true];};
 if (parseNumber(_val) <= 0) exitWith {hint localize "STR_NOTF_enterAmountGive";ctrlShow[2002,true];};
```

Comment:
No need for () around _val, also no need for semi-colon before closing curly bracket.